### PR TITLE
add icons types

### PR DESCRIPTION
**Pull request by dependentmadani**
Trying to solve the issue #44, @neSpecc can you please check this?

### CODE REVIEW

File: dist/index.d.ts
Comment (neSpecc) [inline dist/index.d.ts:1]: why do we need both index.d.ts and types/types.d.ts?

File: README.md [hunk 1]
```diff
@@ -33,83 +33,83 @@ You will get the line: `<svg xmlns="http://www.w3.org/2000/svg" ...`
...
-| ![IconTranslate](dist/icons/IconTranslate.svg) | `IconTranslate` |
-| ![IconTrash](dist/icons/IconTrash.svg) | `IconTrash` |
-| ![IconTwitter](dist/icons/IconTwitter.svg) | `IconTwitter` |
-| ![IconUnderline](dist/icons/IconUnderline.svg) | `IconUnderline` |
-| ![IconUndo](dist/icons/IconUndo.svg) | `IconUndo` |
-| ![IconUnlink](dist/icons/IconUnlink.svg) | `IconUnlink` |
-| ![IconUser](dist/icons/IconUser.svg) | `IconUser` |
-| ![IconUsersGroup](dist/icons/IconUsersGroup.svg) | `IconUsersGroup` |
-| ![IconWarning](dist/icons/IconWarning.svg) | `IconWarning` |
+| ![IconAddBackground](dist\icons\IconAddBackground.svg) | `IconAddBackground` |
```
Comment (neSpecc) [inline README.md:36]: wrong directory separator

image

File: src/types/types.d.ts
Comment (neSpecc) [inline src/types/types.d.ts:1]: why do we need this file also? is it created manually?

**Comment by dependentmadani** [comment]
## Description of new pull request:
- Remove the created types folder and file of every type created manually.
- Add a line in generate.js file to add type for every icon, and it will be build in `index.d.ts`.

File: dist/index.d.ts
Comment (neSpecc) [inline dist/index.d.ts:1]: 1) why `export declare const` is used?
2) why `export declare type` is used? 

I'm not sure, do we need both of them?

**Comment by dependentmadani** [comment]
## Description of last modification:
- Add the export of types in packages.json
- Remove all previous modifications.

**Review by neSpecc — APPROVED** [review]

**Pull request closed without merging**